democracy_club/apps/hermes/templatetags/post_tags.py

```diff
@@ -6,6 +6,7 @@
 
 def posts_for_tag(tag, hide_header=False):
     posts = Post.objects.for_tag(tag)[:4]
+    posts = [p for p in posts if p.is_published]
```

Comment:
If the top post isn't published then this will only return 3 posts, not 4 as expected. 

Rather, we only want to query posts that are published, so can you filter these out on the database site:

`posts = Post.objects.for_tag(tag).published()[:4]`

Reply:
Now fixed.